kolibri/core/assets/src/vue/session-nav-widget/index.vue

```diff
@@ -119,6 +119,9 @@
   $size-sm = 30px
   $border = 2px
 
+  .logicon
+    width: 500px // need to be bigger than actual width to center
```

Comment:
The `500px` thing is really not correct. The problem here is related to the `div` wrapper around the two `nav-bar-item` subcomponents. We'd seen similar behavior before, where the parent element wasn't getting `display: table-cell` as it was meant to because of that wrapper div.

I addressed this in the PR by making `nav-bar-item` the top-level element.